.gitignore

```diff
@@ -1,6 +1,5 @@
 lastCheckTime.txt
 node_modules
-Makefile
```

Comment:
I currently have a Makefile with a series of commands to bring up Simone and revert back to Spike for commit. Should be able to refactor this process easily, though, and will probably have some to add to this PR. That'll probably include support for a `.env.testing` for the Simone token and adding a flag on startup to support that.